frameworks/g-cloud-10/search_mappings/services.json

```diff
@@ -1,7 +1,7 @@
 {
   "settings": {
     "index": {
-      "max_result_window": 20000
+      "max_result_window": 25000
```

Comment:
I'd potentially suggest being a bit more generous. If we double instead, we'll probably not have to look at this again next time :-)

Reply:
Yep, seems reasonable!